os-image/os-image-tracking-table.md

```diff
@@ -0,0 +1,11 @@
+
+#### Release table for Add-server and Repair-server
+
+When you add a machine or repair a machine, make sure to deploy the correct version of OS image to the Azure Local machine. Use the following table to identify the correct OS image version and the download location for your Azure Local machine:
+
+
+| To deploy this OS image on Azure Local machine... | Download OS image from this location |
```

Comment:
Should we also add a link for OS image if using Azure Gov cloud?

Reply:
No, this is only for Add-server and Repair-server issue.